.github/workflows/tests.yml

```diff
@@ -7,7 +7,7 @@ env:
 
 jobs:
   test-action:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
thought: We could specify `ubuntu-24.04` which is the same thing right now instead of `ubuntu-latest`; I don't have strong opinions though:

* `ubuntu-latest` means you shouldn't need to change this again, unless there's some sudden incompatibility with a new version.
* `ubuntu-24.04` means nothing changes without you knowing.

Reply:
I think for something simple and infrequently updated like this, the benefits of not having to update it again outweigh the benefits of keeping the version deterministic.

I think we might make different calls for other repos though.